R/clean_data.R

```diff
@@ -135,7 +135,9 @@ clean_data <- function(data, ...) {
   ## | Column names in 'keep' will not be modified.
   ## -----
   if (!is.null(params[["standardize_column_names"]])) {
-    message("\ncleaning column names\n")
+    cli::cli_alert_info(
+      tr_("\ncleaning column names")
```

Comment:
"Cleaning column names". Use capital letter at start of sentence for consistency with  other messages.